pkg/tracer/tracer_linux.go

```diff
@@ -608,21 +609,20 @@ func (t *LinuxTracer) dispatchPerfEvent(event *common.PerfEvent) {
 				}
 			}
 		}
-		t.tcpConnInsights[httpRes.Connection] = conn
+		t.tcpConnInsights[connection] = conn
 
 	} else if event.MySQLGreeting != nil {
-		logger.Tracef("mysql greeting: %v", event.MySQLGreeting)
-		mysqlGreeting := event.MySQLGreeting
+		logger.Warnf("mysql greeting: %v", event.Connection)
```

Comment:
also trace

Reply:
This was for debugging, it has been reverted to trace in the latest commit